website/docs/r/federated_database_instance.html.markdown

```diff
@@ -0,0 +1,149 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: federated_database_instance"
+sidebar_current: "docs-mongodbatlas-resource-federated-database-instance"
+description: |-
+    Provides a Federated Database Instance resource.
+---
+
+# Resource: mongodbatlas_federated_database_instance
+
+`mongodbatlas_federated_database_instance` provides a Federated Database Instance resource.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+~> **IMPORTANT:** All arguments including the password will be stored in the raw state as plain text. [Read more about sensitive data in state.](https://www.terraform.io/docs/state/sensitive-data.html)
+
+## Example Usages
+
+
+```terraform
+resource "mongodbatlas_federated_database_instance" "test" {
+  project_id         = "PROJECT ID"
+  name = "NAME OF THE FEDERATED DATABASE INSTANCE"
+  aws {
```

Comment:
@andreaangiolillo just FYI i think it's fine for now, but next year this resource will likely be expanded to support Azure cloud too